contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -162,25 +236,25 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
                 }),
                 requiredGasLimit: REPORT_BURNED_TOKENS_REQUIRED_GAS,
                 allowedRelayerAddresses: allowedRelayerAddresses,
-                message: abi.encode(ITokenSource.SourceAction.Burn, abi.encode(totalBurnedTxFees))
+                message: abi.encode(ITokenSource.SourceAction.Burn, abi.encode(scaledAmount))
             })
         );
 
         emit ReportTotalBurnedTxFees({
             teleporterMessageID: messageID,
-            burnAddressBalance: totalBurnedTxFees
+            burnAddressBalance: scaledAmount
```

Comment:
thinking if on the desteination chain we should keep the amounts in event emissions in the denomination of the scaled amount

Reply:
Maybe add both?

Reply:
nvm, will just use the value for this chain.